static/app/components/nav/utils.tsx

```diff
@@ -0,0 +1,170 @@
+import type {ComponentProps} from 'react';
+import type {LocationDescriptor} from 'history';
+
+import type Feature from 'sentry/components/acl/feature';
+import {isItemActive} from 'sentry/components/sidebar/sidebarItem';
```

Comment:
```suggestion
import {isNavItemActive} from 'sentry/components/sidebar/sidebarItem';
```
Given that most things now seem Nav namespaced